host.py

```diff
@@ -321,7 +321,7 @@ def run_on_bf(self, cmd: str, log_level: int = logging.DEBUG) -> Result:
 
     def run_in_container(self, cmd: str, interactive: bool = False) -> Result:
         name = "bf"
-        setup = f"sudo podman run --pull always --replace --pid host --network host --user 0 --name {name} -dit --privileged -v /dev:/dev -v /lib:/lib quay.io/bnemeth/bf"
```

Comment:
By having  /lib here breaks the CDA?

Reply:
Yes, I get a 'moduleNotFound' error when running bf tools (specifically pxeboot looking for pexcept) when I mount lib:lib. I don't believe this is necessary as long as kernel-modules-extra is installed in the host

Reply:
Got it thanks!